torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19794,7 +19794,7 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            sample_inputs_func=sample_inputs_zero_),
     OpInfo('logsumexp',
            aliases=('special.logsumexp',),
-           dtypes=all_types_and(torch.bool, torch.half, torch.bfloat16),
+           dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
```

Comment:
Does this support complex half (`torch.complex32`) if so, it should be added to this list.

Reply:
I get `RuntimeError: "exp_vml_cpu" not implemented for 'ComplexHalf'`. So I guess supporting `complex32` would require some ground work first.